app/controllers/admin/collections_controller.rb

```diff
@@ -209,7 +209,9 @@ def destroy
         target_path = admin_collection_path(@target_collection)
         @source_collection.reload
       else
-        flash[:error] = "Collection contains invalid media objects that cannot be moved. Please address these issues before attempting to delete #{@source_collection.name}."
+        mos = @source_collection.media_objects.to_a
+        invalid_mos = mos.collect { |mo| "<li>#{mo.id}</li>" unless mo.valid? }.compact
+        flash[:error] = "Collection contains invalid media objects that cannot be moved. Please address these issues before attempting to delete #{@source_collection.name}.<ul>#{invalid_mos.join('')}</ul>".html_safe
```

Comment:
I was testing this and am not sure how to skip the validations for this. When you delete the collection it calls `#reassign_media_objects`, which runs into a validation error when attempting to set `media_object.collection = target_collection`. From the stack trace, this seems to be related to the media object model having `has_and_belongs_to_many :governing_policies`. I've tried inserting `validate: false` at various points in the chain but I think it may be something deeper in ActiveFedora itself that is running the validation.

At this point I lean towards nixing this part of the PR since it is out of the scope of the ticket, unless you want to take a look at things.